final_task/rss_reader/ConsoleParse.py

```diff
@@ -0,0 +1,14 @@
+import argparse
+
+
+def get_arguments_from_console():
+    arg_parser = argparse.ArgumentParser(description="Pure Python command-line RSS reader.")
+    group = arg_parser.add_mutually_exclusive_group()
```

Comment:
(Optional) It is better to make that if the source is specified, then application looks through local cache and prints only news from the specified source